src/js/controllers/sidebar.js

```diff
@@ -21,8 +21,12 @@ angular.module('copayApp.controllers').controller('sidebarController',
 
     self.switchWallet = function(wid) {
       self.walletSelection = false;
-      profileService.setAndStoreFocus(wid, function() {});
-      go.walletHome();
+      $rootScope.$emit('Local/SwitchWallet');
```

Comment:
SetAndStoreFocus already fire `Local/NewFocusedWallet` that runs `clearStatus`, so it will
be called 2 times.